modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/frontend/data/set/ClassicSystemFDSEntry.java

```diff
@@ -36,7 +36,7 @@ public String getName() {
 
 	@Override
 	public String getRESTApplication() {
-		return "c/fdssamples";
+		return "/c/fdssamples";
```

Comment:
URL fix, unrelated to the rest of PR. With this, imported SDS can be seen in fragment.